packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../android/aar.dart';
+import '../build_info.dart';
+import '../project.dart';
+import '../runner/flutter_command.dart' show DevelopmentArtifact, FlutterCommandResult;
+import 'build.dart';
+
+class BuildAarCommand extends BuildSubCommand {
+  BuildAarCommand({bool verboseHelp = false}) {
+    addBuildModeFlags(verboseHelp: verboseHelp);
+    usesFlavorOption();
+    usesPubOption();
+    argParser
+      ..addMultiOption('target-platform',
+        splitCommas: true,
+        defaultsTo: <String>['android-arm', 'android-arm64'],
+        allowed: <String>['android-arm', 'android-arm64', 'android-x86', 'android-x64'],
+        help: 'The target platform for which the project is compiled.',
+      )
+      ..addOption('output-dir',
+        help: 'The absolute path to the directory where the repository is generated.'
+              'By default, this is \'<current-directory>android/build\'. ',
+      );
+  }
+
+  @override
+  final String name = 'aar';
+
```

Comment:
report usage for whether it's run on a plugin or module

Reply:
Done \o/